clientd3d/d3drender_textures.h

```diff
@@ -0,0 +1,21 @@
+// Meridian 59, Copyright 1994-2024 Andrew Kirmse and Chris Kirmse.
+// All rights reserved.
+//
+// This software is distributed under a license that is described in
+// the LICENSE file that accompanies it.
+//
+// Meridian is a registered trademark.
+//
+// The render textures code to create from resource (png, bfg - regular or swizzled).
+// This also contains the means to set the palette for these textures.
```

Comment:
Where is that?  There are no functions in this file that mention a palette.  Remove this line?